From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jenna Himawan <jhimawan@google.com>
Date: Mon, 27 Apr 2026 08:25:53 -0700
Subject: [PATCH] [bugfix] upstream: Fix strings in camera site settings

The "Don't allow" radio button in the Camera dialog was incorrectly
labeled "Don't allow sites to see your location". This CL
corrects the string to refer to the camera. It also corrects
makes the "Ask" option plural for greater consistency with the
"Don't allow" option and other site settings' text.

Bug: 499339600
Change-Id: Ie70b6844e4a256805c781049629d2d2a4df83a3e
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/7794657
Reviewed-by: Theresa Wellington <twellington@chromium.org>
Commit-Queue: Jenna Himawan <jhimawan@google.com>
Cr-Commit-Position: refs/heads/main@{#1621104}
---
 components/browser_ui/strings/android/site_settings.grdp      | 4 ++--
 .../IDS_WEBSITE_SETTINGS_CAMERA_ASK.png.sha1                  | 2 +-
 .../IDS_WEBSITE_SETTINGS_CAMERA_BLOCK.png.sha1                | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/components/browser_ui/strings/android/site_settings.grdp b/components/browser_ui/strings/android/site_settings.grdp
index 3c3bf7d324a80..ea1d9eb69ea68 100644
--- a/components/browser_ui/strings/android/site_settings.grdp
+++ b/components/browser_ui/strings/android/site_settings.grdp
@@ -615,10 +615,10 @@
     Sites usually use your video camera for communication features like video chatting
   </message>
   <message name="IDS_WEBSITE_SETTINGS_CAMERA_ASK" desc="Primary text corresponding to the ask button in the camera permission radio button group.">
-    Site can ask for your camera
+    Sites can ask for your camera
   </message>
   <message name="IDS_WEBSITE_SETTINGS_CAMERA_BLOCK" desc="Primary text corresponding to the block button in the camera permission radio button group.">
-    Don't allow sites to see your location
+    Don't allow sites to use your camera
   </message>
   <message name="IDS_WEBSITE_SETTINGS_CAMERA_BLOCK_DESCRIPTION" desc="Description of block camera permission.">
     Features that need a camera won't work
diff --git a/components/browser_ui/strings/android/site_settings_grdp/IDS_WEBSITE_SETTINGS_CAMERA_ASK.png.sha1 b/components/browser_ui/strings/android/site_settings_grdp/IDS_WEBSITE_SETTINGS_CAMERA_ASK.png.sha1
index 10143ea02e369..19cba1e1a9254 100644
--- a/components/browser_ui/strings/android/site_settings_grdp/IDS_WEBSITE_SETTINGS_CAMERA_ASK.png.sha1
+++ b/components/browser_ui/strings/android/site_settings_grdp/IDS_WEBSITE_SETTINGS_CAMERA_ASK.png.sha1
@@ -1 +1 @@
-722c85ab392ac39764276735666cab481d4975fb
\ No newline at end of file
+f37e21ab1098fe0d459226d1f47fc599b6ca4e85
\ No newline at end of file
diff --git a/components/browser_ui/strings/android/site_settings_grdp/IDS_WEBSITE_SETTINGS_CAMERA_BLOCK.png.sha1 b/components/browser_ui/strings/android/site_settings_grdp/IDS_WEBSITE_SETTINGS_CAMERA_BLOCK.png.sha1
index 10143ea02e369..5109217e1bd02 100644
--- a/components/browser_ui/strings/android/site_settings_grdp/IDS_WEBSITE_SETTINGS_CAMERA_BLOCK.png.sha1
+++ b/components/browser_ui/strings/android/site_settings_grdp/IDS_WEBSITE_SETTINGS_CAMERA_BLOCK.png.sha1
@@ -1 +1 @@
-722c85ab392ac39764276735666cab481d4975fb
\ No newline at end of file
+392589b6b55d41d1e11135eb25eb5eb948092258
\ No newline at end of file
